webapi/_test_data/ontologies/something-onto.ttl

```diff
@@ -65,4 +67,6 @@
 
 	knora-base:subjectClassConstraint :Something ;
 
-	knora-base:objectClassConstraint knora-base:LinkValue .
+	knora-base:objectClassConstraint knora-base:LinkValue ;
+
+	salsah-gui:guiElement salsah-gui:Searchbox .
```

Comment:
Could you add a newline at the end of the file?

Reply:
Sorry I missed that. Added the new line right now.

Reply:
Thanks! It shouldn't matter, the triplestore did not complain ;-) It is rather a issue of style.